demo/full/scripts/controllers/ContentList.tsx

```diff
@@ -143,19 +149,18 @@ function constructContentList(): Partial<Record<string, IDisplayedContent[]>> {
     {},
   );
 }
-
 /**
- * Generate URL with hash-string which can be used to reload the page with the
+ * Generate a config object which can be used to reload the page with the
  * current stored content or demo content. This can be used for example to
  * share some content with other people.
- * Returns null if it could not generate an URL for the current content.
- * @param {Object} content - The content object as constructed in the
+ * Returns null if it could not generate a config for the current content.
+ * @param {IPlayableContent} content - The content object as constructed in the
```

Comment:
As `IPlayableContent` is only a TypeScript interface, the general idea in RxPlayer code is just to put `Object` instead in JSDoc

Reply:
JSDoc is not able to take Typescript interface ?

Reply:
No idea.
 
https://jsdoc.app/tags-param

 Tells us:
> The parameter type can be a built-in JavaScript type, such as string or Object, or a [JSDoc namepath](https://jsdoc.app/about-namepaths) to another symbol in your code. If you have written documentation for the symbol at that namepath, JSDoc will automatically link to the documentation for that symbol. You can also use a type expression to indicate, for example, that a parameter is not nullable or can accept any type; see the [@type tag documentation](https://jsdoc.app/tags-type) for details.

So I suppose it depends on their definition of a valid "symbol" here.

Moreover JSDoc is only the "language" which various tools may also parse (including TypeScript which I would guess would understand interfaces) which may or may not understand those - no idea which is why it would probably be better to just stick to the JSDoc specification.